common/models/wordbreakers/src/default/index.ts

```diff
@@ -525,22 +525,22 @@ function findBoundaries(text: string, options?: DefaultWordBreakerOptions): numb
   return boundaries;
 
   ///// Internal utility functions /////
+}
 
-  /**
-   * Returns the position of the start of the next scalar value. This jumps
-   * over surrogate pairs.
-   *
-   * If asked for the character AFTER the end of the string, this always
-   * returns the length of the string.
-   */
-  function positionAfter(pos: number): number {
-    if (pos >= text.length) {
-      return text.length;
-    } else if (isStartOfSurrogatePair(text[pos])) {
-      return pos + 2;
-    }
-    return pos + 1;
+/**
+ * Returns the position of the start of the next scalar value. This jumps
+ * over surrogate pairs.
+ *
+ * If asked for the character AFTER the end of the string, this always
+ * returns the length of the string.
+ */
+export function positionAfter(text: string, pos: number): number {
+  if (pos >= text.length) {
+    return text.length;
+  } else if (isStartOfSurrogatePair(text[pos])) {
+    return pos + 2;
```

Comment:
what happens if pos+2 is off the end of the buffer?

Reply:
fcn may not be needed with other suggestion.

Reply:
> what happens if pos+2 is off the end of the buffer?

This gives the index you'd provide to the `end` param of a substring operation... though admittedly assuming it'll be followed up by the low surrogate, rather than checking.

It's been in place like this for a _while_, admittedly.